### ISSUE TITLE
Выплата аванса по ЗП

### ISSUE THREAD

**Issue by Linda4ka** [Contabilizare/c5#294]
Получается очень интересно:1. Сделала табель с 1.12 по 10.12 по сотрудникам.
2. Начислила зарплату за этот период.
3. Сделала документ Аванс по  ЗП. Тут же столкнулась с тем, что при попытке заполнить документ, программа берет не сумму из начисления, а 50% от оклада. Получается сумма больше той, что в Начислении. Поменяла % вручную, но тут точно не угадаешь и получилось что выплата меньше начисления. Ладно, это можно пережить.
4. Делаем Начисление в конце декабря за месяц. В документе Начисление заполняет закладку Авансы. Вроде все нормально.
5. Делаем документ Выплата ЗП и тут неприятный сюрприз: Документ заполняется окладом 100% без учета выданного раннее аванса. Поменяла % выплата вручную и примерно попала в сумму, необходимую к выплате. 
Если надо, могу скинуть базу чтобы вы могли изучить. Может я недоработала, а может что-то не работает. Сообщите!

**Comment by Nullarity** [comment]
> 2. Начислила зарплату за этот период.

В этом случае, аванса по ЗП уже нет, сотрудник получает деньги за отработанное время. Таким образом, Аванс по ЗП использоваться не должен.

**Comment by Linda4ka** [comment]
Да, но получается что выплата в конце месяца,, за оставшиеся 15 дней считается полным окладом. Мне кажется что это неправильно. Или я ошибаюсь?

**Comment by Nullarity** [comment]
Рассмотрение примера в данном случае не имеет смысла, потому что Авансы по ЗП введены в этой схеме неуместно (ошибочно). Я не отрицаю существование какой-то другой проблемы, но лучше ее рассматривать без участия документа Авансы по ЗП.

**Comment by Linda4ka** [comment]
Тогда я скину ссылку на базу и вы сами посмотрите чтоб решить в чем проблема.
